lib/databases/elasticsearch6.js

```diff
@@ -170,20 +178,115 @@ _.extend(Elasticsearch.prototype, {
 	},
 
 	findOne: function(query, queryOptions, callback) {
-		queryOptions.size = 1;
+    queryOptions.size = 1;
 
 		this.find(query, queryOptions, function(err, vms) {
 			if (err) {
 				return callback(err);
 			}
 
 			if (vms.length === 0) {
-				return callback(null, null);
+        return callback(null, null);
 			}
 			callback(null, vms[0]);
 		});
 	},
 
+  bulkCommit: function (vms, callback) {
+    
+    if (!vms) return callback(new Error());
+
+    var self = this;
+
+    this.checkConnection(function(err) {
+			if (err) {
+				return callback(err);
+			}
+
+      var bulkOperation = [];
+      var vmMap = {};
+
+      var deletedCount = 0;
+      var indexCount = 0;
+
+      _.each(vms, function(vm) {
+        if (!vm.actionOnCommit) return;
+        vmMap[vm.id] = vm;
+        switch(vm.actionOnCommit) {
+          case 'delete':
+            if (vm.version === null) return;
+            bulkOperation.push({ delete: { _index: self.indexAndTypeName, _type: self.indexAndTypeName, _id: vm.id, version: vm.version } });
+            deletedCount++;
+            break;
+          case 'create':
+            obj = vm.attributes;
+            bulkOperation.push(
+              { index: { _index: self.indexAndTypeName, _type: self.indexAndTypeName, _id: vm.id, op_type: 'create' } },
+              obj,
```

Comment:
is there a missin } ?

Reply:
not really, but there is a trailing comma which is a problem on older node